catroid/src/androidTest/java/org/catrobat/catroid/test/common/ActionSchedulerTest.java

```diff
@@ -49,128 +48,114 @@
 @RunWith(AndroidJUnit4.class)
 public class ActionSchedulerTest {
 	private ActionScheduler scheduler;
-	private Action actionMock;
+	private EventThread threadMock;
 	private Actor actor;
 
 	@Before
 	public void setUp() {
 		actor = new Look(new Sprite());
 		scheduler = new ActionScheduler(actor);
-		actionMock = mock(Action.class);
-		when(actionMock.act(anyFloat())).thenReturn(false);
+		threadMock = mock(EventThread.class);
+		when(threadMock.act(anyFloat())).thenReturn(false);
 	}
 
 	@Test
-	public void startActionTest() {
-		when(actionMock.act(anyFloat())).thenReturn(false);
+	public void startThreadTest() {
+		when(threadMock.act(anyFloat())).thenReturn(false);
 
-		scheduler.startAction(actionMock);
+		scheduler.startThread(threadMock);
 		scheduler.tick(1);
 
 		Assert.assertEquals("Unexpected number of actions in scheduler", 1, actor.getActions().size);
-		Assert.assertEquals("Invalid action in scheduler", actionMock, actor.getActions().get(0));
+		Assert.assertEquals("Invalid action in scheduler", threadMock, actor.getActions().get(0));
```

Comment:
Please use static imports [[1]](https://en.wikipedia.org/wiki/Static_import).
And please remove the message from the assert call. 😉